src/services/Peers.js

```diff
@@ -56,5 +58,43 @@ module.exports = {
       return b.public_key > a.public_key ? 1 : -1;
     });
     return peers;
+  },
+
+  mergeOldAndNew: function(newPeers, oldPeers) {
+    var oldPeersByPubKey = {};
+    _.each(oldPeers, function(item) {
+      oldPeersByPubKey[item.new.public_key] = item.new;
+    });
+    var ret = _.map(newPeers, function(p) {
+      return { new: p, old: oldPeersByPubKey[p.public_key]};
+    });
+    return ret;
+  },
+
+  animateChange: function(propertyArray) {
+    function animate($element, oldValue, newValue) {
+      var originalColor = '#333';
```

Comment:
I can, but animate function assumes it gets an element and oldValue, newValue to animate between